utils/config/daos_server.yml

```diff
@@ -177,13 +177,13 @@
 ## Number of hugepages to allocate for DMA buffer memory
 #
 ## Specifies the number (not size) of hugepages to allocate for use by NVMe
-## through SPDK. Note that each target requires 1 GB of hugepage space, so
+## through SPDK. Note that each target requires 1 GiB of hugepage space, so
 ## this value needs to represent the total amount of hugespace required for
 ## all targets across all engines on host, divided by the system hugepage size.
-## If not set here, an appropriate value will be automatically calculated and
-## default system hugepage size will be used.
+## If not set here, an appropriate value will be automatically calculated based
+## on the number of targets (using the default system hugepage size of 2MiB).
 #
-## Example: (2 engines * (8 targets/engine * 1GB)) / 2MB hugepage size = 16834
+## Example: (2 engines * (16 targets/engine * 1GiB)) / 2MiB hugepage size = 16834
```

Comment:
maybe we want to make it a bit more clear, what about
```
system huge page size: 2048 kib == 2048 * 1024 bytes == 2097152 bytes
memory requirement per target (for average workload): 1 gib == 1024 * 1024 * 1024 bytes == 1073741824 bytes
hugepages required per target: 1073741824 / 2097152 == 512 hugepages/target
so for 2 engines with 16 targets each a total of 32 * 512 == 16384 hugepages total is required
```

Reply:
i think  this is perfectly clear as it stands, GiB and MiB are well-defined units (IEC 60027-2). i just wanted to fix the 2x error in the math...